### ISSUE TITLE
Website not visible in google search

### ISSUE THREAD

**Issue by SaleemCastle** [SaleemCastle/next-react-portfolio-website#3]
The site is not being indexed as shown in google console.
Recommendations:
1 - Crawl the website for potential errors including missing alt text, meta descriptions missing
2 - Provide a sitemap for Google to crawl and index correctly

Helpful Resources: 
https://sosteve.medium.com/your-next-js-website-isnt-on-google-time-to-fix-it-c8f92f515e35#:~:text=Fix%20recipe,sitemap%20inside%20Google%20Search%20Console.
